### PR TITLE
fix bug where replacing nodes would only replace every other

### LINKED ISSUE #6: properties are not being copied over correctly
e.g. we're evaporating them hah

### PR DESCRIPTION
**Pull request by reminyborg**
using the testing code from @rreusser i found a major bug in node children replacing.

This should hopefully resolve #6 ✨

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ function updateChildren (newNode, oldNode) {
       }
     } else if (retChildNode !== oldChildNode) {
       oldNode.replaceChild(retChildNode, oldChildNode)
+      iNew--
```
Comment (rreusser) [inline index.js]: Good catch! I felt like I hadn't exhausted all of the possibilities there.

Comment (reminyborg) [inline index.js]: I used way to much time finding it... 😔

Comment (rreusser) [inline index.js]: Sorry about that! Glad to witness the incremental progress though.

Comment (reminyborg) [inline index.js]: No problem! It is so much more rewarding when you find it. ✨

**Comment by yoshuawuyts** [comment]
Tested and works! Don't have internet on my laptop rn but we should merge and publish this ASAP; think this was the last known bug you just solved 🎉🎉🎉

**Comment by yoshuawuyts** [comment]
`v2.1.3` :sparkles:

**Pull request merged**